scala-cli/.SRCINFO

```diff
@@ -0,0 +1,12 @@
+pkgbase = scala-cli
+	pkgdesc = Scala CLI is a command-line tool to interact with the Scala language.
+	pkgver = 0.0.9
+	pkgrel = 1
+	url = https://scala-cli.virtuslab.org
+	arch = x86_64
+	license = Apache
+	depends = java-runtime-headless>=8
+	source = https://github.com/VirtusLab/scala-cli/releases/download/v0.0.9/scala-cli-x86_64-pc-linux.gz
+	sha256sums = 2035cef8c8059674fb54fba0710aff2062d9783fef585a89d03d335486d3b64e
+
+pkgname = scala-cli
```

Comment:
Btw this might be auto-generated. Maybe it can be created as a part of the build process and filtered using .gitignore using:

```
makepkg --printsrcinfo > .SRCINFO
```